sprint-issue-template.md

```diff
@@ -0,0 +1,32 @@
+# Sprint <% date %>
+
+## Sprint Goals
+
+- 
+
+## Sprint Discussions
+
+[Schedule](https://github.com/ipfs/pm#sprint-discussion-schedule)
+
+Please take notes in a separate pad, if you can, and link it here.
+
+Endeavour         | Lead           | Pad    | Permanent Link
+:---------------: | :------------: | :----: |
+Apps on IPFS      | Juan Benet     |        |
```

Comment:
maybe we can break this up into: 
- Station + Webui led by @dignifiedquire
- Apps on IPFS  led by me
?